packages/twenty-front/src/modules/object-record/hooks/useAggregateRecords.ts

```diff
@@ -35,10 +36,16 @@ export const useAggregateRecords = <T extends AggregateRecordsData>({
     recordGqlFieldsAggregate,
   });
 
+  const { objectPermissionsByObjectMetadataId } = useObjectPermissions();
+
+  const hasReadPermission =
+    objectPermissionsByObjectMetadataId[objectMetadataItem.id]
```

Comment:
I wonder if we should replace all these by an util but I'm not sure it would be that nice:
hasReadPermission({ objectPermissionsByObjectMetadataId, objectMetadataId: objectMetadataItem.id)
hasWritePermission...